lib/v2/finology/bullets.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable linebreak-style */
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const baseUrl = 'https://insider.finology.in/bullets';
+
+    const { data: response } = await got(baseUrl);
+    const $ = cheerio.load(response);
+
+    const listItems = $('ul.timeline li.m-pb2')
+        .toArray()
+        .map((item) => {
+            item = $(item);
+            const year = new Date().getFullYear();
+            const time = `${item.find('div.timeline-info span').text().split(', ')[1]} ${year}`;
```

Comment:
There's no need to inject current year. `dayjs`(the backend behind `parseDate`) will handle it.